lmnet/lmnet/datasets/mscoco.py

```diff
@@ -225,10 +225,16 @@ def coco_category_id_to_lmnet_class_id(self, cat_id):
     @functools.lru_cache(maxsize=None)
     def _gt_boxes_from_image_id(self, image_id):
         """Return gt boxes list ([[x, y, w, h, class_id]]) of a image."""
+        classes = set(class_name for class_name in self.classes if class_name is not "__background__")
```

Comment:
`class_ids` should be a set BUT`classes` should be a list.

Reply:
sorry, you right. I fixed this.